### PR TITLE
WASMetadataExtractor is downloaded from jenkins

### LINKED ISSUE #54: WASMetadataExtractor.jar: should not be checked into git
The `WASMetadataExtractor.jar` is checked into `jar/` but in `config/deploy.rb` the deploy process downloads the `WASMetadataExtractor.jar` file from Jenkins.

### PR DESCRIPTION
**Pull request by drh-stanford**
This PR fixes #54

### CODE REVIEW

**Comment by drh-stanford** [comment]
Hmm looks like the *specs* require the jar...

**Comment by drh-stanford** [comment]
the WasMetadataExtractor repo already has tests.

File: spec/wasCrawlPreassembly/lib/metadata_extractor_service_spec.rb [hunk 1]
```diff
@@ -8,60 +8,6 @@
...
 
-  context Dor::WASCrawl::MetadataExtractor, '.call_java_library with druid that has one warc file' do
-    it 'should create a valid output file' do
-      druid_id = 'druid:ab123ab1234'
-      metadata_extractor_service = Dor::WASCrawl::MetadataExtractor.new(@collection_id, @crawl_id, @staging_path.to_s, druid_id)
-
-      metadata_extractor_service.run_metadata_extractor_jar
-
-      output_file = 'tmp/' + druid_id + '.xml'
-      expect(File.exist?(output_file)).to be_truthy
```
Comment (ndushay) [inline spec/wasCrawlPreassembly/lib/metadata_extractor_service_spec.rb]: hrmm.  This test could be morphed to check if the java command to be executed is correct?    That would be a more appropriate test anyway -- the correctness of the output xml should be tested in the java project, no?

Comment (ndushay) [inline spec/wasCrawlPreassembly/lib/metadata_extractor_service_spec.rb]: hrmm. This test could be morphed to check if the java command to be executed is correct? That would be a more appropriate test anyway -- the correctness of the output xml should be tested in the java project, no?

**Comment by drh-stanford** [comment]
ok i changed the test to check the syntax of the java command we run

File: spec/wasCrawlPreassembly/lib/metadata_extractor_service_spec.rb [hunk 2]
```diff
@@ -12,53 +12,8 @@
     it 'should create a valid output file' do
```
Comment (ndushay) [inline spec/wasCrawlPreassembly/lib/metadata_extractor_service_spec.rb]: d'oh - need to change this description

File: spec/wasCrawlPreassembly/lib/metadata_extractor_service_spec.rb [hunk 3]
```diff
@@ -12,53 +12,8 @@
...
       druid_id = 'druid:ab123ab1234'
       metadata_extractor_service = Dor::WASCrawl::MetadataExtractor.new(@collection_id, @crawl_id, @staging_path.to_s, druid_id)
-
+      expect(metadata_extractor_service).to receive(:system).with(%r{java .*-jar jar/WASMetadataExtractor.jar -f XML .*/ab123ab1234/content -o tmp/druid:ab123ab1234.xml -c config/extractor.yml --collectionId test_collection --crawlId test_crawl}).and_return(true)
```
Comment (ndushay) [inline spec/wasCrawlPreassembly/lib/metadata_extractor_service_spec.rb]: maybe put expected java command on separate line for readability?

**Comment by drh-stanford** [comment]
ok updated

**Comment by ndushay** [comment]
Thank you!  Changes much appreciated.

**Pull request merged**